cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1382,10 +1382,10 @@ def _add_launch_templates(self, managed_placement_groups, instance_profiles):
 
             queue_placement_group = None
             if queue.networking.placement_group:
-                if queue.networking.placement_group.id and (
+                if (queue.networking.placement_group.id or queue.networking.placement_group.name) and (
                     queue.networking.placement_group.enabled or queue.networking.placement_group.is_implied("enabled")
                 ):  # Do not use `PlacementGroup/Id` when `PlacementGroup/Enabled` is explicitly set to `false`
```

Comment:
NIT:
We can remove this comment
```
# Do not use `PlacementGroup/Id` when `PlacementGroup/Enabled` is explicitly set to `false`
```
Or update it with
```
# Do not use `PlacementGroup/Id` or `PlacementGroup/Name` when `PlacementGroup/Enabled` is explicitly set to `false`
```

Reply:
will do!